src/apps/activities/tests/fixtures/configs.py

```diff
@@ -179,3 +180,8 @@ def audio_player_config(default_config: DefaultConfig) -> AudioPlayerConfig:
 @pytest.fixture
 def phrasal_template_config(default_config: DefaultConfig) -> PhrasalTemplateConfig:
     return PhrasalTemplateConfig(**default_config.dict(), type=ResponseType.PHRASAL_TEMPLATE)
+
+
+@pytest.fixture
+def unity_config(default_config: DefaultConfig) -> UnityConfig:
+    return UnityConfig(**default_config.dict(), type=ResponseType.UNITY)
```

Comment:
Please add a test, check for test_create_applet_with_each_activity_item it is a parameterized test for all item types